Feature/tune 90 central cms yourwords

RichardLanders opened the pull request.

Commit:
exploring wagtail admin models

Commit:
added wagtailmodeladmin to requirements.txt

Commit:
added django-import-export to requirements.txt

Commit:
added ability to export entries based off filter, entries link to competition now points to wagtail admin yourwords competition

Commit:
removed export url

Commit:
added tests for wagtailmodeladmin

Commit:
flake8 fixes

Commit:
fixed failing test

RichardLanders commented:
@saeedmb8 @miltontony

miltontony commented on molo/yourwords/tests/test_wagtail_admin_action.py:
```diff
@@ -0,0 +1,62 @@
+
+        date = str(datetime.datetime.now().date())
+
+        print response
```
stray print

miltontony commented on molo/yourwords/tests/test_wagtail_admin_action.py:
```diff
@@ -0,0 +1,62 @@
+            '7,{0},1,test,test body,1,1,0,0,0'.format(date)
+        )
+
+        print expected_output
```
stray print

smarzban commented on molo/yourwords/wagtail_hooks.py:
```diff
@@ -0,0 +1,89 @@
+                    'submission_date', 'is_read', 'is_shortlisted',
+                    'is_winner', '_convert']
+
+    list_filter = ['competition__slug']
```
Can we filter here with ['submission_date', 'is_read', 'is_shortlisted', 'is_winner'] instead

Commit:
added additional filters and a better way of handling filtered exports

Commit:
removed prints from tests

Commit:
added date range filter to yourwords model admin

Commit:
flake8 fix

g1039 commented on molo/yourwords/templates/admin/yourwords/yourwords_date_range_filter.html:
```diff
@@ -0,0 +1,63 @@
+    <button type="submit" id="resetbtn">{% trans "Clear" %}</button>
+    </p>
+</form>
+{% endwith %}
```
Missing a blank line at the of the file.

nathanbegbie committed:
Pulled from origin develop and fixed merge conflicts

nathanbegbie committed:
Removed dependency on wagtailmodeladmin to use core version

See http://docs.wagtail.io/en/v1.8/releases/1.5.html#modeladmin

nathanbegbie committed:
Updated styling for competition admin filter

See: https://github.com/praekelt/molo.profiles/pull/50/files

nathanbegbie committed:
Added better admin interface

TODO: In future we should investigate and implement restirctions on what can be implemented

nathanbegbie committed:
Updated .gitgnore to exclude .DS_store files

See: https://github.com/praekelt/molo/blob/develop/.gitignore#L46

nathanbegbie committed:
Implemented custom competition index page

nathanbegbie committed:
Removed 'add' button for Competitions

nathanbegbie committed:
Added missing newline

smarzban commented on molo/yourwords/models.py:
```diff
@@ -117,6 +117,23 @@ class YourWordsCompetitionEntry(models.Model):
+        MultiFieldPanel(
+            [
+                FieldPanel('competition'),
+                # FieldPanel('submission_date'),
```
@nathanbegbie why is this commented out?

Commit:
Pin django_compressor to 2.0

Commit:
Add compress command to travis

Commit:
Update .travis.yml

nathanbegbie committed:
Made changes to which fields are included in the edit panel for entries

nathanbegbie committed:
Merge branch 'feature/TUNE-90-central-cms-yourwords' of github.com:praekelt/molo.yourwords into feature/TUNE-90-central-cms-yourwords

saeedmb8 commented:
👍  🍰🍰🍰🍰🍰🍰🍰
